chips/tm4c129x/src/usb/common/mod.rs

```diff
@@ -0,0 +1,7 @@
+/// Some generally-useful facilities that could perhaps

```

Comment:
I'd recommend you delete the entire `usb` subfolder for now, since it's not actually doing anything

Reply:
Yes, I will delete the entire usb folder.